src/poliastro/util.py

```diff
@@ -71,3 +72,22 @@ def norm(vec):
 
     """
     return np.sqrt(vec.dot(vec))
+
+
+def time_range(start, *, periods=50, spacing=None, end=None):
+    """Generates range of astronomical times.
```

Comment:
This docstring should have `Parameters` and `Returns`.

Reply:
Good point, thanks! Will do.